### DISCUSSION TITLE
Github copilot for business canceled personal copilot license

### DISCUSSION THREAD

**Discussion by acomarce** [community/community#141503] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi,

I had a personal Github copilot subscription which got canceled when my employer purchased a business copilot license, without me being given the option to keep my personal account. For reference I have a personal github account connected to an orgs enterprise account.

This raises a couple of questions:

1. What data and telemetry is shared through the business licensed copilot, and is that data limited to the company repos?
2. Is there a way to keep both subscriptions?

**Comment by qkfang** [comment, marked as answer]
Hey, here is info from github copilot docs:

Q1:
[Enabling or disabling prompt and suggestion collection](https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/managing-copilot-policies-as-an-individual-subscriber#enabling-or-disabling-prompt-and-suggestion-collection)
**You can choose whether your prompts and Copilot's suggestions are collected and retained by GitHub**, and further processed and shared with Microsoft. For more information about data that GitHub Copilot may collect depending on your settings, see "[GitHub Terms for Additional Products and Features](https://docs.github.com/en/site-policy/github-terms/github-terms-for-additional-products-and-features#github-copilot)" and the [GitHub Copilot privacy FAQ](https://github.com/features/copilot/#faq-privacy-copilot-for-business).

https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/managing-copilot-policies-as-an-individual-subscriber

on top of above, i remember enterprise license might have a setting for copilot to collect prompt results to help fine tuning the model.

Q2:
[About changes to your Copilot subscription](https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/managing-your-copilot-subscription/about-billing-for-github-copilot-individual#about-changes-to-your-copilot-subscription)
**If you have an active GitHub Copilot Individual subscription, and are then assigned a seat as part of a GitHub Copilot Business or GitHub Copilot Enterprise subscription, your personal GitHub Copilot subscription will be automatically canceled**. You will receive a prorated refund for any remaining portion of your personal subscription's current billing cycle. You will then be able to continue using GitHub Copilot according to the policies set at the enterprise or organization level.

https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/managing-your-copilot-subscription/about-billing-for-github-copilot-individual

**Comment by Himani0002** [comment]
May I use copilot for free ? please tall me ?
